### ISSUE TITLE
Own caches search should ignore filter

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/cgeo#12724]
### Describe your problem!

Previously (before the new filter implementation) the search for owned caches (in search actionbar) explicitly ignored the setting to hide owned caches as it otherwise would have no results. Now the live filter is applied to this search leading to a possible empty result if the live filter hides owned caches.

I suggest to reestablish the old behavior for this specific search. The feature is designed to offer cache owners a quick access to their owned caches although they normally don`t want them to show up on map or in searches.

### How to reproduce?

- Have a live filter set up to hide owned caches
- Open Search activity
- Tap on "star" icon in actionbar

### Actual result after these steps?

Empty result is shown

### Expected result after these steps?

Still show the owned caches (not apply owned caches filter) for this specific search. This would restore the original behavior.

### Reproducible

Yes

### c:geo Version

2022.02.12-NB

**Comment by Lineflyer** [comment]
In general this was decided, we did IMHO however not take into account this special "Show my owned caches" function (see "How to reproduce" above) which is useless with filtering "owned caches".
I would not want to ignore live filter for any other search, but IMHO it has to be ignored for this special search.

**Comment by eddiemuc** [comment]
I agree with @Lineflyer. In this special case it does not make sense IMHO to use the LIVE filter. Instead a TRANSIENT filter should be applied

**Comment by SammysHP** [comment]
Fixed with #18101.
